pdoc/cli.py

```diff
@@ -401,9 +401,12 @@ def recursive_add_to_index(dobj):
 
     @lru_cache()
     def to_url_id(module):
-        url = module.url()
-        if top_module.is_package:  # Reference from subfolder if its a package
-            _, url = url.split('/', maxsplit=1)
+        # We want to reference the url from the location of search.html. We want to
+        # replace all the already known recrusion - 1
+        # (list index start at 0, so we dont need to change anything)
+        recrusion = top_module.url().count('/')
+        url = '/'.join(module.url().split('/')[recrusion:])
```

Comment:
`recrusion` -> `recursion`
```suggestion
        # replace all the already known recursion - 1
        # (list index start at 0, so we dont need to change anything)
        recursion = top_module.url().count('/')
        url = '/'.join(module.url().split('/')[recursion:])
```